source/qdiscs.rst

```diff
@@ -240,15 +240,32 @@ the qdisc defines how Linux networking stack priorities map into traffic
 classes and how traffic classes map into hardware queues. Besides that, it
 also enables the user to configure the GCL for a given interface.
 
-No NIC driver in kernel mainline currently supports the EST feature so TAPRIO
-hardware offload isn't supported. However, EST can still be leveraged since
-TAPRIO provides a TxTime-assisted implementation (available since kernel 5.3)
-and a pure software implementation. In Tx-Time-assisted mode, the LaunchTime
-feature is used to schedule packet transmissions, emulating the EST feature.
-The NIC must support LaunchTime to be able to use that mode. If not, use the
-pure software implementation. This tutorial uses the Intel(R) Ethernet
-Controller I210 which supports LaunchTime, thereby setting TAPRIO up for
-using the TxTime-assisted mode.
+There are several NIC drivers in mainline Linux that support the EST hardware
+feature. That includes:
+
+* Intel Ethernet controllers like I210 and I225/226
+* Synopsys Ethernet controllers like stmmac
```

Comment:
From what I remember `stmmac` is the name of driver, not the name of the product/device, right? Same for `enetc`. As most of those are embedded in the mainboards, not sure how to best mention them.

Reply:
Yes, these are the Linux driver names. I've updated it.